.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  win: circleci/windows@2.4.0
+  win: circleci/windows@5.0.0
```

Comment:
Looks like cli still supports 2.4.1, so we should also make sure not to break that: https://github.com/snyk/cli/blob/master/.circleci/config.yml#L5

Reply:
That version of the orb has a bug in `nvm` so I need to upgrade it, but it only manifests itself on `prettier` so it should not cause the CLI to fail.